template projects/whatsapp webhook/71324cd2-82c7-4734-9e77-89736378e643.json

```diff
@@ -0,0 +1,205 @@
+        },
+        "79688e50-8550-40b8-afea-3c7497fcb9f3": {
+          "coordinates": [
+            726,
+            76
+          ],
+          "description": "",
+          "params": {},
+          "prototype": "1d2a6438-e4bd-4105-b491-cebbb4cae348",
+          "name": "log"
+        },
+        "c21170a6-2808-4258-83e4-b2172905e53b": {
+          "coordinates": [
+            124,
+            -128.5
+          ],
+          "name": "webhook endpoint",
+          "prototype": "cf29d613-2fc6-4230-8c7f-bd86cb1c4c9e",
+          "params": {}
+        },
+        "165ea6aa-358f-4c07-ac6e-da6d66cfdd60": {
+          "coordinates": [
+            -97,
+            -44.5
+          ],
+          "name": "webhook endpoint params",
+          "prototype": "321d0ab7-fd53-436d-8c08-8eff32103e5f",
+          "params": {
+            "cc0b6ce0-74f3-4f00-982b-d6dba32d5466": "{\n  \"serverId\": \"WhatsApp\",\n  \"port\": 8080,\n  \"method\": \"POST\",\n  \"route\": \"/whatsapp\",\n  \"request\": {\n    \"contentType\": \"json\"\n  }\n}"
+          },
+          "description": ""
+        },
+        "cee21975-3bf9-4386-9fca-e8681e869c4f": {
+          "coordinates": [
+            -112,
+            266.5
+          ],
+          "name": "verify endpoint params",
+          "prototype": "321d0ab7-fd53-436d-8c08-8eff32103e5f",
+          "params": {
+            "cc0b6ce0-74f3-4f00-982b-d6dba32d5466": "{\n  \"serverId\": \"WhatsApp\",\n  \"port\": 8080,\n  \"method\": \"GET\",\n  \"route\": \"/whatsapp\"\n}"
+          }
+        },
+        "9ca28c40-581a-4ba8-bba5-a1556d884d18": {
+          "coordinates": [
+            207,
+            274.5
+          ],
+          "name": "verify endpoint",
+          "prototype": "cf29d613-2fc6-4230-8c7f-bd86cb1c4c9e"
+        },
+        "8ec0bf8c-33a2-48e8-91f0-362fe754a8c1": {
+          "coordinates": [
+            323.33333333333326,
+            186.5
+          ],
+          "name": "handle verify",
+          "prototype": "aa581a6e-e909-470b-9097-1a0fd447248c",
+          "params": {}
+        },
+        "31b8f37b-2cf1-46e3-9975-af333b6f19c5": {
+          "coordinates": [
+            -85,
+            107.5
+          ],
+          "name": "handle verify params - FILL IT!!!",
+          "prototype": "485b6519-9402-4e78-8599-47ad52c92b2f",
+          "params": {
+            "543f8ad2-7755-44f6-bc9b-e6208c87b15a": "{\"verifyToken\": \"\"}"
+          },
+          "description": "Set the verify token of the webhook configuration.\n\n\nExample: \n```\n{\"verifyToken\": \"TOKEN\"}\n```\nMore:  https://developers.facebook.com/docs/graph-api/webhooks/getting-started\n\nHandle the verify token as a secret. \nRecommended solution is storing in evironment variable.\n\nMore: https://docs.cranq.io/toolbox/using-environment-variables"
+        },
+        "739cda90-47ca-4973-931d-c94564bfb0c3": {
+          "coordinates": [
+            304.5,
+            -6.5
+          ],
+          "name": "YOUR LOGIC",
```

Comment:
Maybe name it like "YOUR LOGIC - REPLACE ME!"
To be consistent with "FILL IT" named nodes